core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html

```diff
@@ -7,29 +7,24 @@ <h3 class="activity-title protractor-test-exp-summary-tile-title">
           <span ng-if="isWindowLarge"><[getExplorationTitle()|truncate:40]></span>
           <span ng-if="!isWindowLarge"><[getExplorationTitle()|truncate:40]></span>
         </h3>
-
         <!-- Note that if this is ng-if instead, the avatar area will not detect a mouseover and the tile will behave incorrectly. -->
         <div ng-show="isWindowLarge && avatarsList.length > 0" class="exploration-summary-avatars" ng-show="avatarsList.length > 0">
           <div>
             <circular-image ng-if="avatarsList.length > 0"
-                            src="avatarsList[0].image"
-                            link="avatarsList[0].link"
-                            tooltip="<[avatarsList[0].tooltipText | truncate:14]>"
-                            tooltip-placement="right">
+            src="avatarsList[0].image"
```

Comment:
Align src etc. with the ng-if on the previous line. This makes it easy to see that this is a continuation of the existing tag.